src/goto-programs/goto_coverage.cpp

```diff
@@ -1,7 +1,10 @@
 #include <goto-programs/goto_coverage.h>
 
-size_t goto_coveraget::total_branch = 0;
+size_t goto_coveraget::total_assert = 0;
```

Comment:
Do we really need these global variables?

Reply:
It is the static member, must be defined and initialized once outside the class definition